package-lock.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "jellyfin-apiclient",
+  "version": "0.0.0",
```

Comment:
Shouldn't we version it `10.0.0` for now? cc @nvllsvm

Reply:
I based that on this [comment](https://github.com/jellyfin/jellyfin-web/pull/33#discussion_r247355549), but I can change it if we're good with using `10.0.0`.